pwndbg/aglib/vmmap.py

```diff
@@ -41,3 +42,34 @@ def find(address: int | pwndbg.dbg_mod.Value | None) -> pwndbg.lib.memory.Page |
         return page
 
     return pwndbg.aglib.vmmap_custom.explore(address)
+
+
+def addr_region_start(address: int | pwndbg.dbg_mod.Value) -> int | None:
+    """
+    Let's define a "region" as contiguous memory compromised of memory mappings
+    which all have the same object file name. Also referred to as "File (Base)" by
+    `xinfo`.
+
+    Returns:
+        The start of the memory region this address belongs to, or None if the address
+        is not mapped or a valid memory region couldn't be found.
+    """
+    address = int(address)
```

Comment:
Is int() needed if the arg is supposed to be int anyway? or can it be gdb.Value (our wrapper) or what?:)